CHANGELOG.md

```diff
@@ -3,6 +3,93 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.0.0
+
+_Nov 12, 2021_
+
+🎉 This is the new stable release of the data grid component 🎉!
+
+If you want to migrate your DataGrid or DataGridPro from v4 to v5, take a look at our [Migration Guide](https://mui.com/components/data-grid/migration-v4/).
+This version is fully compatible with `@mui/material@5.X` and can be used with `@material-ui/core@4.x` with some additional steps [described here](https://mui.com/components/data-grid/migration-v4/#using-mui-x-v5-with-mui-core-v4)
+
+A big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- ???
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@v5.0.0` / `@mui/x-data-grid-pro@v5.0.0`
+
+#### Breaking changes
+
+- [DataGrid] The following CSS classes were renamed to follow the internal convention
+
+  - `MuiDataGrid-columnsContainer` was renamed to `MuiDataGrid-columnHeaders`
+  - `MuiDataGrid-columnHeaderWrapper` was renamed to `MuiDataGrid-columnHeadersInner`
+  - The `scroll` class applied to `MuiDataGrid-columnHeaderWrapper` was renamed to `MuiDataGrid-columnHeadersInner--scrollable`
+
+- [DataGrid] `props.components.Checkbox` and `props.componentsProps.checkbox` were renamed `props.components.BaseCheckbox` and `props.componentsProps.baseCheckbox`
+
+  As a start of the effort for #3066, we renamed the slots for the checkbox to clearly indicate that it is a slot for a core component.
+
+  `components` prop change:
+  ```diff
+  <DataGrid
+    components={{
```

Comment:
This and the following diff could be merged to reduce the size of the changelog.